crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -494,18 +498,39 @@ impl TryFrom<&LogletConfig> for LogletConfiguration {
     }
 }
 
+#[derive(
+    Default, Eq, PartialEq, PartialOrd, Ord, Clone, Copy, derive_more::From, derive_more::Into,
+)]
+struct AttemptsCounter(u8);
+
+impl AttemptsCounter {
+    const OLDEST: AttemptsCounter = AttemptsCounter(0);
```

Comment:
This caught my eye on the first pass but I didn't have a better idea - I find `OLDEST` slightly ambiguous here; maybe `EARLIEST` or `INITIAL` makes more sense. Only if you agree, whatever you decide is best.

Reply:
You are right. I will rename it. I think `FIRST` is better name in this context